File: DataBinding/PageAndRDL/ODataDataSource/VB.NET/ObjectDataSourceClient/Models/Movie.vb
Friend Class Movie
	Public Property MovieID() As Integer
	Public Property Title() As String
	Public Property MPAA() As String

	Public Sub New(id As Integer, title As String, mpaa As String)
		Me.MovieID = id
		Me.Title = title
		Me.MPAA = mpaa
	End Sub
End Class

File: DataBinding/PageAndRDL/ODataDataSource/C#/ObjectDataSourceClient/Models/Movie.cs

namespace ObjectDataSource.Models
{
	internal class Movie
	{
		public int MovieID { get; set; }
		public string Title { get; set; }
		public string MPAA { get; set; }

		public Movie(int id, string title, string mpaa)
		{
			MovieID = id;
			Title = title;
			MPAA = mpaa;
		}
	}
}
